openshift-ci/build-root/Dockerfile

```diff
@@ -4,3 +4,4 @@ FROM registry.svc.ci.openshift.org/openshift/release:golang-1.12
 
 RUN yum -y install make wget gcc git httpd-tools
 
+ADD oc /usr/bin/oc
```

Comment:
Can’t we `yum install oc`? So that we are quite sure always that the version added is a stable release?which

Reply:
For stable release version we will already be creating an `oc` binary image separately in multi-stage https://github.com/openshift/release/pull/9431/files#diff-f04f6304bab9d5afb9c78addf7eb17a2R18 which I think will take care of this version thing.

Reply:
BTW we are using the corresponding `oc` version with respect to the cluster version through our job config file . For example 4.2 cluster will be accessed through 4.2 oc and same is applicable to other cluster version too. 
Yes, it is possible to install the version of oc through the docker file but it will unnecessarily increase the complexity of the file. Just to avoid such complexity here we are using a prow provided mechanism of oc binary copy through the job config.
There is no such direct mechanism to inject corresponding `oc` binary into test container PATH. So as per the change i made in https://github.com/openshift/release/pull/9431/files#diff-f04f6304bab9d5afb9c78addf7eb17a2R16, i need to copy the oc binary into the test container PATH. So, i added the line ``ADD oc /usr/bin/oc`` here in the docker file